interactive_ai/workflows/train/trainer/pyproject.toml

```diff
@@ -2,7 +2,7 @@
 name = "otx-v2-trainer"
 version = "1.0.0"
 description = "OTX trainer"
-requires-python = ">=3.10, <3.11"
+requires-python = ">=3.11, <=3.12"
```

Comment:
Why this change? At the moment Geti uses Python 3.10 everywhere. Is DEIM incompatible with py3.10?

Reply:
OTX now doesn't support 3.10 version. We switched to 3.11 - 3.12 support

Reply:
Resolved